sdk/core/Azure.Core/tests/public/ModelSerializationTests/Models/Animal.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core.Serialization;
+
+namespace Azure.Core.Tests.Public.ModelSerializationTests
+{
+    public class Animal : IUtf8JsonSerializable, IModelJsonSerializable<Animal>
```

Comment:
Curious: is there a timeline for removing the IUtf8JsonSerializable interface?

Reply:
no timeline, but will come after we migrate everything which could be a long tail for hand written libraries